app/src/main/res/navigation/nav_graph.xml

```diff
@@ -587,7 +587,15 @@
 
     <fragment
         android:id="@+id/aboutFragment"
-        android:name="org.mozilla.fenix.settings.about.AboutFragment" />
+        android:name="org.mozilla.fenix.settings.about.AboutFragment">
```

Comment:
Connect the layout file to improve the preview pane
```suggestion
        android:name="org.mozilla.fenix.settings.about.AboutFragment"
        tools:layout="@layout/fragment_about_libraries">
```

Reply:
This one should be `fragment_about` right?